real_estate/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import estate_property
+from . import estate_property_type
+from . import estate_property_tag
+from . import estate_property_offer
```

Comment:
```suggestion
from . import estate_property_offer
from . import estate_property_tag
from . import estate_property_type
```
Try to maintain an alphabetical order